.env.docker_example

```diff
@@ -0,0 +1,16 @@
+NODE_ENV=development
```

Comment:
Why do you have 2 new example env files? Docker and Local should use the same env.

Reply:
When I'm running locally mongo is on localhost and so is redis. @crablar  had he same setup. I figured it was fairly common.

Reply:
Yea, but you should still be able to use the same env values for docker. If not or you aren't using docker, you simply need to change that in your .env file. We shouldn't have extra files for env examples

Reply:
The difference when I use local vs docker is just this:

+MONGO_HOST=mongodb://**mongo**/express-mongoose-es6-rest-api-development

vs
+MONGO_HOST=mongodb://**localhost**/express-mongoose-es6-rest-api-development


.env.example and .env.docker_example are copies though.


Either way, I am fine with removing them and just adding a note in the tips and tricks part of the docs.

Reply:
I guess if you the mongo url is required we should keep it. I figured localhost would work on docker

Reply:
It would be nice if they were the same :) Maybe that's possible. I'm removing the old .env.example for clarity.